xbmc/Application.h

```diff
@@ -168,7 +168,7 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
   CApplicationStackHelper& GetAppStackHelper();
   void OnPlayBackEnded() override;
   void OnPlayBackStarted(const CFileItem &file) override;
-  void OnPlayerCloseFile(const CFileItem &file, const CBookmark &bookmark) override;
+  void OnPlayerCloseFile(const CFileItem &file, const CBookmark &bookmark, const StreamsInfoPtr streams = nullptr) override;
```

Comment:
1) no default parameters
2) this is a generic callback, not a video player specific one, hence no stream info here